department-of-veterans-affairs/caseflow
# TYLERB/APPEALS-11755: Remove last name warning for unrecognized claimants in HLR/SC for listed attorneys

Resolves [APPEALS-11755](https://vajira.max.gov/browse/APPEALS-11755)

### Description
Changed the Add Claimant Confirmation Modal to not display the last name warning for known listed attorneys due to the way the text label is parsed into title, first name, middle name, and last name.

### Acceptance Criteria
- [x] The Add Claimant Confirmation Modal does not display the information warning for no last name for a known listed attorney for Higher Level Reviews
- [x] The Add Claimant Confirmation Modal does not display the information warning for no last name for a known listed attorney for Supplemental Claims
- [x] The Add Claimant Confirmation Modal does not display the information warning for no last name for a known listed attorney for 10182 Appeals
- [x] The warning still displays correctly for other individual claimants that are missing last name in 10182 appeal decision reviews

### Testing Plan
1. Select an intake user BVAISHAW  will work
2. Start a HLR or SC intake and select an unknown claimant on the review page
3. Select attorney as the relationship type
4. Enter a name and select one of returned attorneys from the dropdown as the claimant
![image](https://user-images.githubusercontent.com/109369527/203163343-103bd447-a5ea-41cd-8f89-47e17f19d71f.png)

5. After the information is inserted into the page click the continue to next step button
6. Verify that the last name information warning is gone from the modal
![image](https://user-images.githubusercontent.com/109369527/203163557-0f5662cf-632f-4e7e-ad98-13938af74a13.png)

7. Cancel or complete that intake
8. Return and start a new appeal intake
9. Select an attorney and check the information warning again to verify that it is also gone in 10182 forms.
10. Remove the attorney and select a child
11. Only Fill in the fields that are required and omit the last name field. (You will have to select a State even though it is marked as optional)
12. Click the continue to next step button
13. Verify that the last name information warning is present in the modal.
![image](https://user-images.githubusercontent.com/109369527/203164029-8a7459d1-3d24-4691-895f-fdd03468d7fb.png)
